client/state/stats/selectors.js

```diff
@@ -0,0 +1,15 @@
+import { getCompositeKey, normalizeParams } from './utils';
+
+export function getStatsItem( state, params ) {
+	const _params = normalizeParams( params );
+	const key = getCompositeKey( _params );
```

Comment:
Note to self: try out the [deterministic-stringify](https://github.com/Automattic/wp-calypso/tree/8761dbc6ee7288a57c89e236d2b406187a6d4079/client/lib/deterministic-stringify) lib here.